<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockLightProperties</class>
 <widget class="QDockWidget" name="DockLightProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>217</width>
    <height>321</height>
   </rect>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
  </property>
  <property name="windowTitle">
   <string>Light Properties</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Wavelength</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="cbIR">
         <property name="text">
          <string>IR (disp as brown)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbRed">
         <property name="text">
          <string>Red</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbYellowBlack">
         <property name="text">
          <string>Yellow (disp as black)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbYellow">
         <property name="text">
          <string>Yellow</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbGreen">
         <property name="text">
          <string>Green</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbBlue">
         <property name="text">
          <string>Blue</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbUV">
         <property name="text">
          <string>UV (disp as violet)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Image Field</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Half Field of View(deg)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="leHalfFOV">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Nb Intermediates angles</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="spNbAngles">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="value">
            <number>3</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
